Marlin/Configuration_adv.h

```diff
@@ -2269,6 +2273,7 @@
     #define X_MICROSTEPS     16    // 0..256
     #define X_RSENSE          0.11
     #define X_CHAIN_POS      -1    // <=0 : Not chained. 1 : MCU MOSI connected. 2 : Next in chain, ...
+    //#define X_INTERPOLATE true      // Use interpolation on X axis 
```

Comment:
If interpolation is supposed to be disabled by commenting out this configuration line, then there's no reason to have it translate to a boolean value.

Reply:
No. If the line is commented out it inherits the value from the global `INTERPOLATE`, whether true or false. Uncommenting the line allows overriding the default.